client/components/share/google-plus-share-preview/style.scss

```diff
@@ -0,0 +1,78 @@
+.google-plus-share-preview {
+	background-color: #fefefe;
+	border-radius: 2px;
+	box-shadow: 0 1px 4px 0 rgba(0,0,0,0.14);
+	color: rgba(0, 0, 0, 0.87);
+	font-family: Roboto, RobotoDraft, Helvetica, Arial, sans-serif;
+	margin: 0 auto;
+	max-width: 530px;
+	width: 100%;
+	
+	// Nesting to win main.scss
+	.google-plus-share-preview__profile-name {
+		color: rgba(0, 0, 0, 0.87);
+
+		&:hover {
+			color: rgba(0, 0, 0, 0.87);
+		}
+	}
+
+	.google-plus-share-preview__article-url {
+		color: rgba(0, 0, 0, 0.87);
+
+		&:hover {
+			color: rgba(0, 0, 0, 0.87);
+		}
+	}
+}
+
+
+.google-plus-share-preview__header {
+	align-items: center;
+	display: flex;
+	padding: 16px;
+}
+
+.google-plus-share-preview__profile-picture-part {
+	flex: 0 0 36px;
+}
+.google-plus-share-preview__profile-picture {
+	border-radius: 50%;
+	display: block;
+	height: 36px;
+	width: 36px;
+}
+
+.google-plus-share-preview__profile-line {
+	font-size: 14px;
+	font-weight: 500;
+	line-height: 1.285714286;
+	margin-left: 6px;
+}
+
+.google-plus-share-preview__message,
+.google-plus-share-preview__article-summary {
+	font-size: 14px;
+	line-height: 20px;
+	margin: 0 16px 16px;
+	word-break: break-word;
+}
+
+.google-plus-share-preview__article-title {
+	border-top: 1px solid #ebebeb;
```

Comment:
mmm, maybe this preview stuff won't be shared but what could be shared is the _meaning_ of the color. There is a rule for this: 
> Don't use custom colors, always utilize what _colors.scss provides. If you have to set a color, use lowercase hex values, and shorten them to their smallest expression (like #aaa).

reference: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#the-dry-principle